src/query/sql/src/executor/physical_plans/physical_hash_join.rs

```diff
@@ -82,9 +82,9 @@ pub struct HashJoin {
     pub enable_bloom_runtime_filter: bool,
     // Under cluster, mark if the join is broadcast join.
     pub broadcast: bool,
-    // Original join type. Left/Right single join may be convert to inner join
-    // Record the original join type and do some special processing during runtime.
-    pub original_join_type: Option<JoinType>,
+    // When left/right single join converted to inner join, record the original join type
+    // and do some special processing during runtime.
+    pub single_to_inner: Option<JoinType>,
```

Comment:
`single_to_inner` is more like a bool type which represents if single to inner join has happened.

Reply:
Yes, we also need to record the original join type.